cookbook/build-admin-frontend.rst

```diff
@@ -31,7 +31,7 @@ Solution 2: Build manually with docker
 
 .. code-block:: bash
 
-    docker run --rm --interactive --tty --volume ${PWD}:/var/project node:14.16.0 /bin/bash
+    docker run --rm --interactive --tty --volume ${PWD}:/var/project node:20-alpine3.18 /bin/sh
```

Comment:
I would not add `-alpine` as they might sometimes make problems so I would keep with the default image and the `/bin/bash`